styles/globals.scss

```diff
@@ -14,6 +14,7 @@
   --light-blue: #64afe97b;
   --green: #0a950ad6;
   --scrollbar-thumb-color: #d3d3d3;
+  --no-emoji-color: #d3d3d3;
```

Comment:
no need to create a new color. Use existing ones

Reply:
Ok